dongbin-book/11-Greedy/01.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @name 모험가 길드
+ * @link 동빈북 11-1
+ * @date 2021-10-01
+ * @ref
+ * @author sunmon
+ */
+
+const readConsole = require("../../lib/utils").readConsole;
+
+const generator = function* () {
+  const n = yield;
+  const input = yield;
+  const arr = input
+    .split(" ")
+    .map((i) => Number.parseInt(i))
+    .sort((a, b) => (a < b ? -1 : a > b ? 1 : 0));
```

Comment:
```suggestion
    .map((i) => Number.parseInt(i))
    .sort((a, b) => (a-b));
```

sort 함수에서 a-b라고만 해줘도 같은 결과가 나오는걸루 알고있는데 저렇게 비교해주신 이유가 궁금합니다 !!

Reply:
sort를 저렇게 쓰는게 버릇이어서 저렇게 작성했습니다 ㅋㅋㅋ